icekit/bin/manage.py

```diff
@@ -7,10 +7,15 @@
 
 
 if __name__ == "__main__":
-    if not os.environ.get('ICEKIT_PROJECT_DIR'):
+    ICEKIT_PROJECT_DIR = os.environ.get('ICEKIT_PROJECT_DIR')
+    if not ICEKIT_PROJECT_DIR:
         print("'ICEKIT_PROJECT_DIR' is not defined.", file=sys.stderr)
         exit(1)
 
+    # Add project dir to Python path
+    if ICEKIT_PROJECT_DIR not in sys.path:
+        sys.path.insert(0, ICEKIT_PROJECT_DIR)
```

Comment:
Python should be able to import anything from current working dir normally, but `manage.py` is found in `PATH` and can be executed from anywhere now. Might be that `manage.py` is being executed from a different CWD by supervisor or some other circumstances? Either way, probably wouldn't hurt to ensure any Python packages that aren't directly being installed are always importable?